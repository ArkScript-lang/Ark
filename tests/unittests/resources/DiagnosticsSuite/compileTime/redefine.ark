(mut a 5)
(let a 6)
