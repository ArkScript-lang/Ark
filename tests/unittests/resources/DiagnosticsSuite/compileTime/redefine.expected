At a @ 1:0
    1 | (mut a 5)
      | ^
    2 | (let a 6)
    3 |
        MutabilityError: Can not use 'let' to redefine variable `a'
